cardano-node/src/Cardano/Node/Tracing/Tracers/P2P.hs

```diff
@@ -1422,6 +1432,10 @@ docInboundGovernor peerAddr = Documented
   --     (InboundGovernor.TrUnexpectedlyFalseAssertion protoIGAssertionLocation)
   --     []
   --     ""
+  -- ,  DocMsg
+  --     (InboundGovernor.TrInboundGovernorError protoSomeException)
+  --     []
+  --     ""
```

Comment:
I don't think this should be commented out, the same for the above `TrUnexpectedlyFalseAssertion`.  Could you uncomment both?

Reply:
I will, sorry about the delay